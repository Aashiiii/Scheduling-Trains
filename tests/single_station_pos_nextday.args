Waverton Gymea "18:54:27"
