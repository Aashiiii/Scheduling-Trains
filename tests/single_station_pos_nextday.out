The next train to Gymea from Waverton departs at 03:45:43
